R/xml_interventions_define.R

```diff
@@ -30,39 +32,35 @@ defineVaccine <- function(baseList, vaccine_parameterization, append = TRUE,
         subPopRemoval = list(afterYears = "5")
       )
     )
-
-    componentData <- vaccine_parameterization[[k]]
-
-    baseList <- .xmlAddList(
-      data = baseList,
+    print(paste0("Writing intervention cohort component ",paste0(k, "_intervention_cohort")," to baseXML file..."))
```

Comment:
I am not a big fan of simply printing output messages. Most of this program is supposed to run via batch script on a HPC and thus, the user will not see these messages anyway. We should implement a verbosity and logging system/options for that.